src/modules/count-active-task.js

```diff
@@ -1,12 +1,14 @@
-import renderList from './renderlist';
+//  import { getList } from './request-list';
 
-function countActiveTask(list, listContainer) {
+const countActiveTask = async (list) => {
+  // const list = await getList(listUrl) || [];
   const countList = list.filter((item) => (!item.done));
+  const showCount = ((countList.length === 0) ? 'No tasks today!' : (`You have ${countList.length} tasks today!`));
```

Comment:
`const showCount = !countList.length ? 'No tasks today!' : `You have ${countList.length} tasks today!`;`